### PR TITLE
Add TimeFromDate ⏰📅

### PR DESCRIPTION
**Pull request by Robindiddams**
**Description:**
Because of how numbers work we can get the timezone without a possible error! Wild!

### CODE REVIEW

File: datetime/time.go [hunk 1]
```diff
@@ -361,6 +361,17 @@ func NewDateFromEpoch(epoch int64) Date {
...
 
+// TimeFromDate returns a time.Time for a date
+func TimeFromDate(date Date) time.Time {
+	ts := DateFromEpoch(date.Epoch)
+	if ts.IsZero() {
+		return ts
+	}
+	// apply timezone
+	loc := time.FixedZone("pinpt", int(date.Offset*60))
+	return ts.In(loc)
```
Comment (jhaynie) [inline datetime/time.go:372]: what in the world? the timezone will come back as "pinpt"

Comment (Robindiddams) [reply datetime/time.go:372]: far as I can tell its just a name i can make it blank, but I wanted to have some fun 🤷‍♀️

Comment (jhaynie) [reply datetime/time.go:372]: i thiink you want this

```
loc := time.FixedZone(ts.Location().String(), int(date.Offset*60))
```

**Review by jhaynie — APPROVED** [review]
📆

**Pull request merged**